flink-core/src/main/java/org/apache/flink/api/common/functions/RuntimeContext.java

```diff
@@ -101,9 +101,9 @@
 
 	/**
 	 * For system internal usage only. Use getAccumulator(...) to obtain a
-	 * accumulator. Use this as read-only.
+	 * accumulator. The returned map must NOT be modified.
 	 */
-	HashMap<String, Accumulator<?, ?>> getAllAccumulators();
+	Map<String, Accumulator<?, ?>> getAllAccumulators();
```

Comment:
The comment says system-internal usage only, but I couldn't find any usages with IntelliJ. Should we remove this then? I guess that this has been replaced by the getSnapshot method of the registry.